src/Filter/Type/DateFilterType.php

```diff
@@ -5,17 +5,27 @@
 namespace araise\TableBundle\Filter\Type;
 
 use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 class DateFilterType extends DatetimeFilterType
 {
+    protected $locale;
+
+    public function __construct(?string $column = null, array $joins = [], protected ?RequestStack $requestStack = null)
+    {
+        parent::__construct($column, $joins);
+        $this->locale = $requestStack->getMainRequest()?->getLocale() ?? 'en';
+    }
+
     public function getValueField(?string $value = null, ?string $operator = null): string
     {
         $date = \DateTime::createFromFormat(static::getQueryDataFormat(), (string) $value) ?: new \DateTime();
         $value = $date->format(static::getDateFormat());
 
         return sprintf(
-            '<input type="date" name="{name}" value="%s">',
-            $operator !== static::CRITERIA_IS_EMPTY ? $value : ''
+            '<input type="date" name="{name}" value="%s" data-controller="araise--core-bundle--datetime" data-araise--core-bundle--datetime-lang-value="%s">',
```

Comment:
Can this be moved to Twig?

Reply:
@renestalder Technically yes, but Symfony itself on other types also isn't doing that for input or select fields.
I see the issue with writting Stimulus controllers/values which aren't readable. But that's how messy Stimulus is without a helper function. Since Symfony has this helper functions for TWIG, we should also be able to access it here in PHP. I'll have a look into that.

Reply:
Had no time to look into that. Made a note for later for myself

Reply:
@marcwieland95 you could use the [StimulusHelper](https://github.com/symfony/stimulus-bundle/blob/2.x/src/Helper/StimulusHelper.php)